apps/remote_control/lib/lexical/remote_control.ex

```diff
@@ -179,7 +179,9 @@ defmodule Lexical.RemoteControl do
         |> Enum.map(&String.trim/1)
 
       {key, value}
+     _ -> nil
```

Comment:
The format looks weird. Can you format it?

Reply:
Ironically I was editing with lexical enabled and it doesn't auto-format on write!

Reply:
Although I've run `mix format` and the formatting is the same

Reply:
auto-format is a client-side feature, not a server-side feature.

Reply:
The formatting is kinda funny, I don't often find case for using multiple `fn` heads in an `Enum` function, but that is indeed the correct syntax and formatting, as far as I can tell

Reply:
it should be 

```elixir
diff --git a/apps/remote_control/lib/lexical/remote_control.ex b/apps/remote_control/lib/lexical/remote_cont
rol.ex
index 99a500a..509f67b 100644
--- a/apps/remote_control/lib/lexical/remote_control.ex
+++ b/apps/remote_control/lib/lexical/remote_control.ex
@@ -170,13 +170,17 @@ defmodule Lexical.RemoteControl do
     env
     |> String.trim()
     |> String.split("\n")
-    |> Enum.map(fn "export " <> key_and_value ->
-      [key, value] =
-        key_and_value
-        |> String.split("=", parts: 2)
-        |> Enum.map(&String.trim/1)
+    |> Enum.map(fn
+      "export " <> key_and_value ->
+        [key, value] =
+          key_and_value
+          |> String.split("=", parts: 2)
+          |> Enum.map(&String.trim/1)
 
-      {key, value}
+        {key, value}
+
+      _ ->
+        nil
     end)
   end
  ```
  
  Formatted by lexical-ls | `mix format`

Reply:
that's not right. the `{key, value}` tuple is the return value of the first clause, not a separate clause on its' own. The `_` clause is the only other clause.

I did try putting the `export " <> key_and_value` part on its' own line and the formatter was fine with it, but it was also fine with it on the same line, so I left it.

Reply:
@scottming I think i've figured out what's up here: there was an issue with `remote_control`'s `.formatter.exs` where it wasn't joining the current path with the impossible to format paths, so the formatter wasn't removing those files. I was running `mix format` from the top level dir, so I didn't see the error until I ran `mix format` from inside the `remote_control` directory. I've updated the PR to fix it